themes/dark/terminal.css

```diff
@@ -1,4 +1,4 @@
-/* [THEME META] {"ver":1, "author":"donut28", "description":"Feel like a hacker with this customised dark theme."} */
+/* [THEME META] {"ver":1, "author":"donut28", "description":"Feel like a hacker with this customised dark theme.",  "accessibility":"optimized"} */

```

Comment:
Accessibility optimized themes are reviewed to a higher standard than the default accessibility level. They should be free of _all_ accessibility issues (up to AAA) on all states (hover, active, etc)
![image](https://user-images.githubusercontent.com/31492625/202985161-2bb2a866-4ab5-4642-a4be-5e91387eeb96.png)
Further review has been paused until this instance is resolved, or until accessibility is set back to standard. 
```suggestion
/* [THEME META] {"ver":1, "author":"donut28", "description":"Feel like a hacker with this customised dark theme."} */
```